frontend/src/pages/modelServing/screens/global/EmptyModelServing.tsx

```diff
@@ -13,16 +20,21 @@ const EmptyModelServing: React.FC = () => {
 
   if (servingRuntimes.length === 0) {
     return (
-      <EmptyState>
-        <EmptyStateIcon icon={PlusCircleIcon} />
+      <EmptyState variant={EmptyStateVariant.small}>
+        <EmptyStateIcon icon={WrenchIcon} />
         <Title headingLevel="h2" size="lg">
-          No model servers
+          No deployed models yet
         </Title>
         <EmptyStateBody>
-          Before deploying a model, you must first configure a model server.
+          To get started, deploy a model from the <strong>Models and model servers</strong> section
+          of a project.
         </EmptyStateBody>
-        <Button variant="primary" onClick={() => navigate('/projects')}>
-          Create server
+        <Button
+          variant="link"
+          onClick={() => navigate('/projects')}
+          style={{ marginTop: 'var(--pf-global--spacer--xl)' }}
+        >
+          Select a project
         </Button>
```

Comment:
If placed within the secondary actions container you get the margin automatically.
See http://v4-archive.patternfly.org/v4/components/empty-state#examples
```suggestion
        <EmptyStateSecondaryActions>
          <Button variant="link" onClick={() => navigate('/projects')}>
            Select a project
          </Button>
        <EmptyStateSecondaryActions>
```

Also, IMO all links navigational links should be rendered as an anchor tag (use react router `Link`) with default link behavior instead of as a button. The user could then use default behavior interactions such as right clicking on the to perform an action as well as shift/cmd clicking.

Reply:
I was thinking the same, but I saw a [similar example](https://github.com/opendatahub-io/odh-dashboard/blob/94a421f3e5f142bedd439de56a17bd414639134f/frontend/src/concepts/pipelines/content/tables/pipeline/PipelinesTableExpandedRow.tsx#L61) in the repository, so I got confused.

Reply:
I was thinking the same, but I saw a [similar example](https://github.com/opendatahub-io/odh-dashboard/blob/94a421f3e5f142bedd439de56a17bd414639134f/frontend/src/concepts/pipelines/content/tables/pipeline/PipelinesTableExpandedRow.tsx#L61) in the repository, so I got confused.

Reply:
@lucferbux  Let's not worry about navigation links being buttons or anchors. I'll create a follow up issue to track a more broader change.

The example you point to also uses `EmptyStateSecondaryActions` :)

Reply:
Ok perfect, yes, that's exactly what I needed, it stills add the 32px margin, thanks @christianvogt we keep the link and we might wanna refactor that in the future.

<img width="371" alt="Screenshot 2023-10-10 at 13 03 36" src="https://github.com/opendatahub-io/odh-dashboard/assets/16117276/3adbab8a-da2e-4d13-84fa-ef924ffe9048">